nautobot/core/graphql/generators.py

```diff
@@ -163,9 +164,15 @@ def generate_list_resolver(schema_type, resolver_name):
     def list_resolver(self, info, **kwargs):
         if schema_type._meta.filterset_class:
             fsargs = {key: [value] for key, value in kwargs.items()}
-            return schema_type._meta.filterset_class(
+            resolved_obj = schema_type._meta.filterset_class(
                 fsargs, model.objects.restrict(info.context.user, "view").all()
-            ).qs.all()
+            )
+            if resolved_obj.errors:
+                errors = {}
+                for key in resolved_obj.errors:
+                    errors[key] = resolved_obj.errors[key]
+                raise GraphQLError(errors)
+            return resolved_obj.qs.all()
```

Comment:
Great to see such a simple fix!

Can you please add a comment to this block about what is going on here, similar to the description you provided in this PR?

Also, what is the behavior that occurs? This is the first place where `raise GraphQLError` is being used in the code base.

Reply:
Expected outcome for the raise has been added to the PR description. When the error is raise the message is sent back as the GraphQL response.